babel.config.js

```diff
@@ -5,7 +5,10 @@ module.exports = {
   ],
   plugins: [
     [require.resolve('@babel/plugin-transform-runtime'), { corejs: 3 }],
-    [require.resolve('babel-plugin-transform-inline-environment-variables')],
+    [
+      require.resolve('babel-plugin-transform-inline-environment-variables'),
+      { include: ['AUTH0_DOMAIN', 'AUTH0_CLIENT_ID'] },
```

Comment:
Curious what problem this caused? :upside_down_face:

Reply:
I was testing dates and one approach was to set process.env.TZ to UTC. 
`babel-plugin-transform-inline-environment-variables` was throwing an error. Not needed anymore.